code401challenges/Data-Structures/src/main/java/Data/Structures/Library.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package Data.Structures;
+
+public class Library {
+    public static class LinkedList {
+        Node head; // Head of the linked list
+
+        LinkedList() {
+            this.head = null;
+        }
+
+        public class Node {
+            int value;  // The value for the Node
+            Node next;  // Reference to the next node in the list
+
+
+            //Constructor
+            Node(int data) {
+                value = data; //Assign the value that is passed
+                next = null;  // Point next node null
+
+            }
+
+        }
+
+        public void insert (int value) {
+            Node newNode = new Node(value);
+
+            if (head == null) {
+                head = newNode;  // new node becomes the head
+            } else {
+                Node temp = head;
+                head = newNode;
+                newNode.next = temp;
+            }
+
+        }
+
+
+        public boolean includes (int value) {
+            Node temp = head;
+
+            while(temp != null) {
+                if(temp.value == value) {
+                    return true;
+                }
+                temp = temp.next;
+
+            }
+            return false;
+
+        }
+
+
+        public String toString() {
+            Node temp = head;
+            String returnString = "";
+            while (temp != null) {
+                returnString += Integer.toString(temp.value) + " ";
+                temp = temp.next;
+            }
+            if (returnString.isEmpty()) {
+                returnString = "{}";
+            }
+            return returnString;
+        }
+    }
+
+    public static void main (String[] args) {
```

Comment:
this is a good start to testing while you write, but does not count towards the grade for testing. I would love to see some well built and running tests in your test file